packages/client/components/AnalyticsPage.tsx

```diff
@@ -70,15 +71,18 @@ const AnalyticsPage = () => {
     }
   )
   const atmosphere = useAtmosphere()
-  useEffect(() => {
+
+  const initLogRocket = async () => {
     const logRocketId = window.__ACTION__.logRocket
     const errorProneAt = window.localStorage.getItem(LocalStorageKey.ERROR_PRONE_AT)
     const expiredErrorProne =
-      errorProneAt && new Date(parseInt(errorProneAt)) < new Date(Date.now() - ms('30d'))
-    if (expiredErrorProne) {
+      errorProneAt && new Date(parseInt(errorProneAt)) < new Date(Date.now() - ms('14d'))
+    const email = window.localStorage.getItem(LocalStorageKey.EMAIL)
+    const res = await fetchQuery<AnalyticsPageQuery>(atmosphere, query, {})
+    const isWatched = res?.viewer?.isWatched
```

Comment:
I noticed that in dev, after updating `isWatched` in the db, it seems to require a `yarn dev` before updating on the client. I updated the caching to `force: true` which didn't do the trick. Perhaps this is expected?

Reply:
ahhh, _just_ for the User table we do some light caching in redis. that'll go away once we go full postgres